packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -50,7 +50,13 @@ const configProvider = (settings: Record<string, ConfigTypes>): IConfigProviderB
 	getRawConfig: (name: string): ConfigTypes => settings[name],
 });
 
-function makeTestContainerConfig(registry: ChannelFactoryRegistry): ITestContainerConfig {
+function makeTestContainerConfig(
+	provider: ITestObjectProvider,
+	registry: ChannelFactoryRegistry,
+): ITestContainerConfig {
+	// When running in cross client compat mode, disable returning local Ids for attachment blobs
+	// because that is not supported by 1.x.
+	const noLocalIdsForAttachmentBlobs = provider.type === "TestObjectProviderWithVersionedLoad";
```

Comment:
I think you do not want to exclude all of TestObjectProviderWithVersionedLoad, just the pairs that have 1.x on either side. I believe it does test N & N-1 pairs, where N is a minor version.

Reply:
Good point. Updated it to only exclude 1.x.